engine/testutil/nodes.go

```diff
@@ -347,23 +343,23 @@ func CollectionNode(t *testing.T, hub *stub.Hub, identity *flow.Identity, rootSn
 	}
 }
 
-// CollectionNodes returns n collection nodes connected to the given hub.
-func CollectionNodes(t *testing.T, hub *stub.Hub, nNodes int) []testmock.CollectionNode {
-
-	colIdentities := unittest.IdentityListFixture(nNodes, unittest.WithRole(flow.RoleCollection))
-	// add some extra dummy identities so we have one of each role
-	others := unittest.IdentityListFixture(5, unittest.WithAllRolesExcept(flow.RoleCollection))
-
-	identities := append(colIdentities, others...)
-	root := unittest.RootSnapshotFixture(identities)
-
-	nodes := make([]testmock.CollectionNode, 0, len(colIdentities))
-	for _, identity := range colIdentities {
-		nodes = append(nodes, CollectionNode(t, hub, identity, root))
-	}
-
-	return nodes
-}
+//// CollectionNodes returns n collection nodes connected to the given hub.
```

Comment:
Why is this commented out?

Reply:
This is not used anywhere, and previously it was calling `CollectionNode` which now depends on some real info(private keys) which makes me question if I should fix it or just leave as it is till we need it.

Reply:
I would remove it if it isn't used